crates/egui-wgpu/src/lib.rs

```diff
@@ -121,7 +121,7 @@ async fn request_adapter(
                 );
             }
 
-            WgpuError::NoSuitableAdapterFound("`request_adapters` returned `None`".to_owned())
+            WgpuError::NoSuitableAdapterFound(format!("`request_adapters` returned `Err`: {}", e))
```

Comment:
Perhaps it would be better to do this...?

```rust
pub enum WgpuError {
    #[error(transparent)]
    NoSuitableAdapterFound(#[from] wgpu::RequestDeviceError),
```

But then that would spread into `NativeAdapterSelectorMethod`...

Reply:
I solved that now by exposing this as a separate error 🤷